services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/system_monitor/_disk_usage.py

```diff
@@ -170,18 +166,12 @@ async def _monitor(self) -> None:
                 msg = f"Could not assign {disk_usage=} for {folder_names=}"
                 raise RuntimeError(msg)
 
-        detected_items = set(usage.keys())
-        if not detected_items.issubset(_SUPPORTED_ITEMS):
-            msg = (
-                f"Computed {usage=}, has unsupported items {detected_items=}. "
-                f"Currently only  the following are supported: {_SUPPORTED_ITEMS}"
-            )
-            raise RuntimeError(msg)
+        supported_usage = {k: v for k, v in usage.items() if k in _SUPPORTED_ITEMS}
 
         # notify only when usage changes
-        if self._last_usage != usage:
-            await self._publish_disk_usage(usage)
-            self._last_usage = usage
+        if self._last_usage != supported_usage:
```

Comment:
Did you consider that `supported_usage` can be empty?

Reply:
The tests currently cover this eventuality. A key of type `MountPathCategory.HOST` is always present.